types/places.js/index.d.ts

```diff
@@ -14,7 +14,8 @@ export = places;
 declare function places(options: places.PlacesStaticOptions): places.PlacesAutocomplete;
 
 declare namespace places {
-    enum PlaceType {
+    // tslint:disable-next-line:no-const-enum
+    const enum PlaceType {
```

Comment:
This shouldnt be enum.. This should be union of type literals instead

Reply:
Thanks @sheetalkamat, should be fixed now.